### ISSUE TITLE
Raspberry Pi CM4 consumes too much power after shutting down

### ISSUE THREAD

**Issue by Minauras** [balena-os/balena-raspberrypi#739]

**Comment by floion** [comment]
Hi,
what version of the Raspberry Pi OS version have you tested with? What kernel version?

**Comment by Minauras** [comment]
Hi,
That's RaspiOS buster armhf lite, release 2021-05-07, Linux kernel 5.10.17

**Comment by floion** [comment]
Have this been observed on the CM4 IO Board? Or on what hardware?

**Comment by Minauras** [comment]
This has been observed on the TOFU board by Oratek.
The setup was identical for both trials, same CM4, and same TOFU board, only the OS changed

**Comment by Minauras** [comment]
Yes exactly, that is raspberrypicm4-ioboard-2.85.2+rev3-v12.10.10
